plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1196,6 +1199,8 @@ public void testJoinPushdown()
                 "nation_lowercase",
                 "AS SELECT nationkey, lower(name) name, regionkey FROM nation")) {
             for (JoinOperator joinOperator : JoinOperator.values()) {
+                log.info("Testing joinOperator=%s", joinOperator);
```

Comment:
This pollutes the logs and makes it much harder to spot where the actual errors are happening. We're actually trying to reduce logging in tests. Please revert this.

Reply:
if there is a failure on CI, how do we know what failed?

Reply:
https://github.com/trinodb/trino/pull/20335

Reply:
You can add a `.describedAs(...)` to the assertion containing the details of which join operator is being evaluated. That will be included in the failure message.